var/spack/repos/builtin/packages/tensorflow-estimator/package.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class TensorflowEstimator(Package):
```

Comment:
Should we rename this package to `py-tensorflow-estimator`?

Reply:
That's fine with me